SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
+===
+
+The bzz protocol consists of a single message type; handshake.
+
+The bzz handshake is the first message to be exchanged between peers
```

Comment:
Might want to be more explicit by "after p2p ...negotiated", what does this map to into terms of referenced devp2p messages?

Reply:
How about "after p2p transport handshake has been completed"?

>  into terms of referenced devp2p messages

Not sure if I understand this question. Any Swarm devp2p protocols can commence immediately after devp2p handshake has been completed.